docs/GettingStartedDocs/buildandsign.md

```diff
@@ -123,6 +122,9 @@ Any properties set in the code also serve as default values when the enclave is
 signed using oesign, so the signing `CONFFILE` only needs to specify override
 parameters during signing.
 
+Using the `OE_SET_ENCLAVE_SGX` macro in code is not recommended and it
```

Comment:
I would suggest removing this line since it's just muddies the waters. Until we have a firm deprecation plan for this feature, it just creates FUD, and when we do have a deprecation plan, we would update the docs around the associated scenarios around this macro (e.g. what is the new recommended way to easily run unsigned debug SGX enclaves?) It's also possible that we will retain backwards compatibility for this, even though it will become the less preferred option.

Right now it just reads like a block of "why are you telling me all this and then also telling me not to use the feature, and then also that I _have_ to use the equivalent for OP-TEE?"

Reply:
Done